src/processTargets/targets/InteriorTarget.ts

```diff
@@ -32,4 +38,34 @@ export default class InteriorTarget extends BaseTarget {
       fullInteriorRange: this.fullInteriorRange,
     };
   }
+
+  createContinuousRangeTarget(
+    isReversed: boolean,
+    endTarget: Target,
+    includeStart: boolean,
+    includeEnd: boolean
+  ): Target {
+    if (isSameType(this, endTarget)) {
+      const constructor = Object.getPrototypeOf(this).constructor;
+
+      return new constructor({
+        ...this.getCloneParameters(),
+        isReversed,
+        fullInteriorRange: createContinuousRangeFromRanges(
+          this.fullInteriorRange,
+          endTarget.fullInteriorRange,
+          includeStart,
+          includeEnd
+        ),
```

Comment:
The issue was that we were letting the base target construct the range, but base target is ignorant of `fullInteriorRange` so it just ended up spreading it from the start of the range

Reply:
I would argue that we don't want full interior range in this case. If you make arrange from one inside to the other I would assume that that only used the content range.

Reply:
Functionally speaking, what's the difference? See the test: it's not including whitespace

Reply:
Then what's the point?

Reply:
I guess if you do a chuck there will a difference.

Reply:
That is probably the test to add

Reply:
btw take tests are so hard to read. Why not use change?

Reply:
Yeah good call to add a "chuck" test

InteriorRange ignores contentRange so I have to use the full range

I used "take" because "change" would have resulted in a fairly unclear result imo. Ie where are the spaces and parens coming from?

Reply:
I understand. A proper test fewer can't come fast enough imo.